packages/sdk/src/RequestController.ts

```diff
@@ -17,11 +18,16 @@ export class RequestController {
 	private basePath: string
 	private headers: Record<string, any>
 	private request: AxiosInstance
+	private apiLimits: ApiLimits
 
 	constructor({ basePath, baseURL, headers }: RequestControllerOptions) {
 		this.basePath = basePath
 		this.baseURL = baseURL
 		this.headers = headers
+		this.apiLimits = {
```

Comment:
Please call this `lastResponseHeaders` and just assign all of the headers from most recent request to this variable so in the future we might also extract other useful informations